docs/refs/eam.bib

```diff
@@ -1101,3 +1101,64 @@ @article{holtslag_preface_2006
         journal = {Boundary-Layer Meterology},
         year = {2006},
 }
+
+@article{blackburn_APE_context_2013,
+	title = {Context and {Aims} of the {Aqua}-{Planet} {Experiment}},
+	volume = {91A},
+	issn = {2186-9057},
+	doi = {10.2151/jmsj.2013-A01},
+	journal = {Journal of the Meteorological Society of Japan. Ser. II},
+	author = {BLACKBURN, Michael and HOSKINS, Brian J.},
+	month = oct,
+	year = {2013},
+	keywords = {precipitation, circulation models, comparison of atmospheric general, gcms, intertropical convergence zone, itcz, modelling hierarchy, tropical circulation},
+	pages = {1--15},
+}
+
+@article{wing_rcemip1_2018,
+	title = {Radiative–convective equilibrium model intercomparison project},
+	volume = {11},
+	issn = {1991-959X},
+	doi = {10.5194/gmd-11-793-2018},
+	abstract = {RCEMIP, an intercomparison of multiple types of models configured in radiative–convective equilibrium (RCE), is proposed. RCE is an idealization of the climate system in which there is a balance between radiative cooling of the atmosphere and heating by convection. The scientific objectives of RCEMIP are three-fold. First, clouds and climate sensitivity will be investigated in the RCE setting. This includes determining how cloud fraction changes with warming and the role of self-aggregation of convection in climate sensitivity. Second, RCEMIP will quantify the dependence of the degree of convective aggregation and tropical circulation regimes on temperature. Finally, by providing a common baseline, RCEMIP will allow the robustness of the RCE state across the spectrum of models to be assessed, which is essential for interpreting the results found regarding clouds, climate sensitivity, and aggregation, and more generally, determining which features of tropical climate a RCE framework is useful for. A novel aspect and major advantage of RCEMIP is the accessibility of the RCE framework to a variety of models, including cloud-resolving models, general circulation models, global cloud-resolving models, single-column models, and large-eddy simulation models.},
```

Comment:
can you delete the abstract from each of these?  Keeps this file from getting to long.

Reply:
done.